src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -90,30 +95,48 @@ public class SkriptCommand implements CommandExecutor {
 		if (TestMode.DEV_MODE)
 			SKRIPT_COMMAND_HELP.add("test");
 	}
-	
-	private static void reloading(CommandSender sender, String what, Object... args) {
+
+	private static void reloading(CommandSender sender, String what, RedirectingLogHandler logHandler, Object... args) {
 		what = args.length == 0 ? Language.get(CONFIG_NODE + ".reload." + what) : Language.format(CONFIG_NODE + ".reload." + what, args);
-		Skript.info(sender, StringUtils.fixCapitalization(m_reloading.toString(what)));
+		String message = StringUtils.fixCapitalization(m_reloading.toString(what));
+		Skript.info(sender, message);
+
+		// Log reloading message
+		if (SkriptConfig.sendReloadingInfoToOps.value())
+			logHandler.log(new LogEntry(Level.INFO, Language.format(CONFIG_NODE + ".reload." + "player reload", sender.getName(), what)));
```

Comment:
does this still log? it would also send to the command sender. it might be better to send this message when building the recipients list.